imports/ui/case-explorer/case-explorer.jsx

```diff
@@ -88,7 +89,7 @@ class CaseExplorer extends Component {
           {!isLoading && Object.keys(unitsDict)
             .reduce((all, unitTitle) => {
               const isExpanded = expandedUnits.includes(unitTitle)
-              const allCases = unitsDict[unitTitle].cases
+              const allCases = unitsDict[unitTitle]
```

Comment:
It looks like you've overridden newer code with the previous code during merge. I think it would be broken now